hws/src/dr_context.py

```diff
@@ -92,6 +93,9 @@ def tree_print(self, verbosity, tabs):
         for t in sorted(self.tables):
             _str = _str + t.tree_print(verbosity, tabs)
 
+        for t in sorted(self.action_ste_tables):
```

Comment:
Let's remove it, it breaks topology
this STE's will be visible from the rules if we jump to them.